### DISCUSSION TITLE
Create beautiful screenshots of snippets

### DISCUSSION THREAD

**Discussion by antonreshetov** [massCodeIO/massCode#112] [Ideas]
What if add the ability to take beautiful screenshots of snippets, like carbon, etc. to the app. What do you think?

![image](https://user-images.githubusercontent.com/17480987/177819863-d1532eaf-da5a-4816-8cb2-108f70b7f675.png)

**Comment by HeliusHao** [comment]
I like, support

**Comment by antonreshetov** [comment]
Done in [v2.8.0](https://github.com/massCodeIO/massCode/releases/tag/v2.8.0)

![demo](https://user-images.githubusercontent.com/17480987/178013116-f22883b3-47f2-4446-963b-ac27aba94f36.gif)

**Reply by ZGGSONG** [reply]
MacOS comes with a window screenshot (Command + Shift + 4 + space). The picture automatically expands a circle of background transparent filling, and adds a layer of shadow around the picture at the same time. You can try it. The effect is really great. If this software can adapt, it is excellent  
like this:
![demo](https://reorx.com/blog/reinitiate-rss-reader/images/netnewswire-1.png)
